src/components/app/App.js

```diff
@@ -4,12 +4,12 @@ import Content from './content/Content';
 import './App.css'
 
 function App() {
-  return (
-    <div className="container">
-      <Header />
-      <Content />
-    </div>
-  );
+    return (
+        <>
```

Comment:
This is a bad practice to leave undefined tags. If you're using it just as a wrapper tag - just use <React.Fragment> instead.